store/store.go

```diff
@@ -246,6 +248,22 @@ func DeleteTeamSecret(c context.Context, s *model.TeamSecret) error {
 	return FromContext(c).DeleteTeamSecret(s)
 }
 
+func GetGlobalSecretList(c context.Context) ([]*model.TeamSecret, error) {
+	return GetTeamSecretList(c, globalTeamName)
+}
+
+func GetGlobalSecret(c context.Context, name string) (*model.TeamSecret, error) {
+	return GetTeamSecret(c, globalTeamName, name)
+}
+
+func SetGlobalSecret(c context.Context, s *model.TeamSecret) error {
+	return SetTeamSecret(c, s)
```

Comment:
is the team secret key set to `__global__` when this function is invoked? or does it need to be set here?